pkgs/applications/window-managers/sway/default.nix

```diff
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   name = "sway-${version}";
-  version = "0.14.0";
+  version = "0.15.0";
```

Comment:
@dywedir You should also update wlc to 0.0.11 in this PR.
> We always depend on the latest version of wlc (master). We ship sway and wlc releases together.
via https://github.com/swaywm/sway/pull/1456#issuecomment-342142666

Reply:
@gnidorah thanks!